projects/openssl/Dockerfile

```diff
@@ -17,6 +17,7 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 RUN apt-get update && apt-get install -y make
 RUN git clone --depth 1 https://github.com/openssl/openssl.git
+RUN git clone --depth 1 --branch OpenSSL_1_1_1-stable https://github.com/openssl/openssl.git openssl111
```

Comment:
Sorry I'm not sure what you mean? @jonathanmetzman may be able to answer questions regarding CIFuzz, I've never used it myself.

Reply:
I mean, every time when I push something on my own git, this fuzzer seems to run a Docker file, that does these
git clone commands, from a fixed repository which is not mine.  So I would like to pass some settings, like
which git repository to clone here:
`2022-11-12T13:25:13.3169491Z ##[command]/usr/bin/docker run --name d5581f791212908fc498b99cea17858d0b2d9_c569bd --label 8d5581 --workdir /github/workspace --rm -e "INPUT_OSS-FUZZ-PROJECT-NAME" -e "INPUT_DRY-RUN" `
so the oss-fuzz-project-name and the dry-run seem to be passed from here
```
      with:
        oss-fuzz-project-name: 'openssl'
        dry-run: false`
```
seem to be passed as environment values, right?
I wonder if we could add another environment which replaces the string `OpenSSL_1_1_1-stable`
here with something else if it is present.

Reply:
I think I understand what you are asking for. You want CIFuzz to work on your own repo rather than only own OpenSSL's?

Reply:
Thanks,, meanwhile I found a way how to work around this, because my feature branch is a work in progress PR in
the openssl repository, all commit hashes are automatically visible in the openssl github repository, and I simply
use the "on: [pull_request]" instead of "on: [push], now can see the result in the workflow of the corresponding PR.